vnext/src/RNTester/ThemingExample.uwp.tsx

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+/* tslint:disable */
+
+import React = require('react');
+import { Text, View, Button } from 'react-native';
+import { AppTheme } from '../../src/index.uwp';
+
+class ThemeExample extends React.Component {
+  state = {
+    currentTheme: AppTheme.currentTheme
+  };
+
+  componentDidMount() {
+    AppTheme.addListener('appThemeChanged', this.onAppThemeChanged);
+  }
+
+  componentWillUnmount() {
+    AppTheme.removeListener('appThemeChanged', this.onAppThemeChanged);
+  }
+
+  onAppThemeChanged = (event: any) => {
+    const currentTheme = AppTheme.currentTheme;
+    this.setState({currentTheme});
+  };
+
+  _onPress = () => {
+  }
+
+  public render() {
+    return (
+      <View>
+        <Text style={{color: 'red'}}>currentTheme: {this.state.currentTheme}</Text>
+        <Button onPress={ this._onPress } title={this.state.currentTheme === 'dark' ? 'Dark' : 'Light'} color={this.state.currentTheme === 'dark' ? 'grey' : 'orange'}></Button>
```

Comment:
>onPress={ this._onPress }  [](start = 16, length = 26)

This doesn't do anything. Does a button require an onPress handler? If not, remove it.